index.js

```diff
@@ -153,3 +153,17 @@ addEventListener('load', () => {
     document.getElementById(`button-${i}`).addEventListener('click', openPopup);
   }
 });
+
+const form = document.getElementById('form');
+const error = document.getElementById('error');
+const email = document.getElementById('email');
+
+form.addEventListener('submit', (event) => {
```

Comment:
- [ ] Great job so far. according to the guidlines, kindly ensure that the error message appears beside the submit button. this will make the notification to be easily noticed.

![Screenshot (446)_LI](https://user-images.githubusercontent.com/65298048/189482206-58ae01eb-77f2-4e17-ae50-e533fb00a1cf.jpg)